package.json

```diff
@@ -33,8 +33,8 @@
   "peerDependencies": {
     "patternfly-react": "2.x",
     "prop-types": "15.x",
-    "react": "16.5.x",
-    "react-dom": "16.5.x"
+    "react": "16.3.x",
```

Comment:
hi @suomiy - are you running into challenges running React `16.5.x` against `patternfly-react`? If that is the case, it would be nice to bubble these issues upstream. Otherwise, what is the problem w/ `16.5`?

Reply:
There is no problem with 16.5. We just need to allign it with https://github.com/openshift/console/blob/master/frontend/yarn.lock because these react versions are incompatible and can crash.